02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                "summary": "It is recommended to collect basic information about non-applicant household members, even if the client does not want to include them on the application, in order to avoid delays and facilitate troubleshooting."
+            },
+            "Who are mandatory HH members for food stamps?": {
+                "score_sum": 0.7800342323376975,
+                "derived_questions": [
+                    "If they do have to list them, do we need to include the adult child's income?"
+                ],
+                "entire_card": "Who are mandatory HH members for food stamps?\nAnswer:\nThe following household members are ALWAYS included, even if they report that they do not share/prepare meals together. (see text only below, see flow chart further down)\nSpouses:\nIf living in the same residence\nChildren:\nIf age 17 or younger, the child is mandatory with legal guardian\nIf age 18-21, the child is mandatory if natural, adopted, or step child*\nA mandatory child's own child or spouse would need to be included on the application as well\n*Ex. If a 20 year old child resides with their parent, they would be mandatory.\n*Ex. If a 20 year old child resides with their grandparent, they would NOT be mandatory. They would likely be sharing/preparing meals together, but are not required to be listed on the same SNAP application.\n\ufeff\n\ufeff",
+                "summary": "The client's 20-year-old son who is living with them and has his own job and income must be included on the application for food stamps, as he is considered a mandatory household member."
```

Comment:
**Relevance to the question**: 🟢 Directly relevant

**Correctness as a summary of the Guru card**: 🟡 Missing Info
- The summary directly answers the original question but doesn't summarize the explanation included in the Guru card about why the son is a mandatory household member. 

**Helpfulness**: 🟢 Helpful